sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/SclService.java

```diff
@@ -512,14 +510,14 @@ public static SclRootAdapter importSTDElementsInSCD(@NonNull SclRootAdapter scdR
             );
 
             if (!mapICDSystemVersionUuidAndSTDFile.containsKey(icdSysVerUuid))
```

Comment:
```suggestion
PrivateService.PrivateLinkedToSTDs pToStd = pToStds
                    .stream()
                    .filter(pTStd -> PrivateService.extractCompasICDHeader(pTStd.tPrivate()).map(TCompasICDHeader::getICDSystemVersionUUID).orElseThrow().equals(icdSysVerUuid))
                    .findFirst()
                    .orElseThrow(() -> new ScdException("There is no STD file found corresponding to " + stdCheckFormatExceptionMessage(tPrivate)));
            // import /ied /dtt in Scd
            SCL std = pToStd.stdList().get(0);
```

Reply:
Linked to change suggestion you made, so application is to be rechecked